packages/core/contracts/financial-templates/perpetual-multiparty/PositionManager.sol

```diff
@@ -459,6 +436,7 @@ contract PositionManager is FeePayer, AdministrateeInterface {
      */
     function redeem(FixedPoint.Unsigned memory numTokens)
         public
+        notEmergencyShutdown()
```

Comment:
Playing devil's advocate: since we allow users to settle post-shutdown, should we also allow them to redeem? @mrice32 @daywiss would like more opinions here

Reply:
My reasoning in this change was that this is simpler & emergency shutdown does not happen often. Am happy to set it up to be similar to the original EMP but emergency shutdown should be far less common then contract expiry.